1518.py

```diff
@@ -0,0 +1,16 @@
+class Solution:
+    def numWaterBottles(self, numBottles: int, numExchange: int) -> int:
+        total, left_empty, to_drink = 0, 0, numBottles
+
+        while to_drink != 0:
+            # Drink and count.
+            drunk = to_drink
+            total += drunk
+
+            # Compute how many new drinks we can get.
```

Comment:
"Compute" is too generic. How about "update total number of drinks available for the next round"?